src/cdk/overlay/position/flexible-connected-position-strategy.ts

```diff
@@ -1204,6 +1217,24 @@ function extendStyles(destination: CSSStyleDeclaration,
   return destination;
 }
 
+/** Shallow-extends a stylesheet object with another stylesheet object, but schedules the process */
+function extendStylesScheduled(
```

Comment:
We can probably keep the same name as the old function unless it was part of the public api